requirements.txt

```diff
@@ -6,3 +6,4 @@ pytest-ansible==2.0.2
 unittest2==1.1.0
 flake8==3.5.0
 fabric==2.5.0
+testimony==2.1.0
```

Comment:
no need to stick to this version, use latest always

Reply:
makes sense, will update pr.